front_matter/doc_editing_setup.md

```diff
@@ -1,4 +1,6 @@
-# Documentation
+# Documentation Editing Setup
+
+The documentation uses the [MyST](https://myst-parser.readthedocs.io/en/latest/) markup language. 
```

Comment:
I wonder which one is the main website to reference, the one you have here or https://mystmd.org/? I think the one you have here might be the one for the Sphinx and Docutils extension to parse MyST, but not the one of MyST itself.

Reply:
Changed.